var/spack/repos/builtin/packages/py-configparser/package.py

```diff
@@ -29,16 +29,13 @@ class PyConfigparser(PythonPackage):
     """This library brings the updated configparser from Python 3.5 to
     Python 2.6-3.5."""
 
-    homepage = "https://pypi.python.org/pypi/configparser"
-    url      = "https://pypi.python.org/packages/source/c/configparser/configparser-3.5.0.tar.gz"
+    homepage = "https://docs.python.org/3/library/configparser.html"
+    url      = "https://pypi.io/packages/source/c/configparser/configparser-3.5.0.tar.gz"
 
-    version('3.5.0', 'cfdd915a5b7a6c09917a64a573140538',
-            url="https://pypi.python.org/packages/7c/69/c2ce7e91c89dc073eb1aa74c0621c3eefbffe8216b3f9af9d3885265c01c/configparser-3.5.0.tar.gz")
+    version('3.5.0', 'cfdd915a5b7a6c09917a64a573140538')
 
-    depends_on('python@2.6:2.8,3.4:')
```

Comment:
Theoretically. I've seen a lot of bugs when packages use things like:
```python
depends_on('py-foo', when='^python@:2.6')
```
meaning that we have to either not support those dependencies (what we did previously, which is why it broke for you) or unconditionally add them for all versions. If I add:
```python
depends_on('python@:3.4.99')
```
it may be difficult for other packages to depend on this package and still build with Python 3.6. But if you want to test it out, let me know if it works for various versions of Python.